subsys/bluetooth/host/smp.c

```diff
@@ -243,6 +249,7 @@ static struct bt_smp_br bt_smp_br_pool[CONFIG_BT_MAX_CONN];
 #endif /* CONFIG_BT_BREDR */
 
 static struct bt_smp bt_smp_pool[CONFIG_BT_MAX_CONN];
+static bool bondable = !!BT_SMP_AUTH_BONDING_FLAGS;
```

Comment:
`static bool bondable = IS_ENABLED(CONFIG_BT_BONDABLE);` might be more readable?